### DISCUSSION TITLE
Keyboard Shortcut

### DISCUSSION THREAD

**Discussion by 3-0-1-2** [Prolo1/Character-Morpher#11] [Ideas]
Would it be possible to add a customizable hotkey combination that can enable and disable the plugin (or sub options)?
Even just a simple button (instead of having to switch to the settings to toggle it on or off) would be really convenient while card browsing or editing.

**Comment by Prolo1** [comment]
there is a keyboard toggle that you can set in the newest version. I added it a couple of updates ago so you may not have noticed. go into "settings" then go to "plugin settings" then find "Character Morpher" and you can change it.

![Screenshot 2023-02-02 133326](https://user-images.githubusercontent.com/100950917/216418186-6c4d1d80-8448-4229-87a2-3528ee15731d.png)

I have it default as RShift+Return(Enter) but you can change it there

**Comment by Prolo1** [comment]
if you have any more problems/questions just message me again and I'll see what I can do.

**Comment by Prolo1** [comment]
I'm also going to put this under the discussions tab to keep myself organized.
